src/Generator/Driver.cs

```diff
@@ -23,6 +23,7 @@ public class Driver
         public Project Project { get; private set; }
         public BindingContext Context { get; private set; }
         public Generator Generator { get; private set; }
+        public bool GenerateSequentialLayout { get; set; } = true;
```

Comment:
What is this new property for? We already have DriverOptions.GenerateSequentialLayout.